lib/bcdice/game_system/DarkDaysDrive.rb

```diff
@@ -69,8 +65,23 @@ def result_2d6(total, dice_total, _dice_list, cmp_op, target)
 
       def eval_game_system_specific_command(command)
         roll_tables(command, TABLES) ||
+          command_iax(command) ||
           RTT.roll_command(randomizer, command)
       end
+
+      private
+
+      def command_iax(command)
+        return nil unless command == "IAX"
+
+        ia = TABLES["IA"].choice(@randomizer.roll_d66(D66SortType::ASC))
+        m = ia.body.match(/\((.+?)\)/)
```

Comment:
```suggestion
        m = ia.body.match(/\(([A-Z]+)\)/)
```
より細かい指定にした方が良さそうに思えました